src/Meziantou.Framework.NuGetPackageValidation/Rules/SymbolsValidationRule.cs

```diff
@@ -237,7 +245,8 @@ await using (pdbData.ConfigureAwait(false))
 
                                 try
                                 {
-                                    using var response = await ShareHttpClient.Instance.GetAsync(uri, HttpCompletionOption.ResponseHeadersRead, context.CancellationToken).ConfigureAwait(false);
+                                    using var request = new HttpRequestMessage(HttpMethod.Get, uri);
+                                    using var response = await context.SendHttpRequestAsync(request, context.CancellationToken).ConfigureAwait(false);
                                     if (!response.IsSuccessStatusCode)
                                     {
                                         context.ReportError(ErrorCodes.UrlIsNotAccessible, $"Source file '{url}' is not accessible", fileName: item);
```

Comment:
Would've been useful for this error to also include the HTTP status code in the reported error message.

Reply:
Yep, you can include it in your PR!

Reply:
Done with 3f3522116022b756c286a1598ae677cdd2234261. Thanks.